src/Model/Resolver/EntityUrl.php

```diff
@@ -139,7 +139,13 @@ public function resolve(
                 $collection = $this->productCollectionFactory->create()
                     ->addAttributeToFilter('status', ['eq' => Status::STATUS_ENABLED]);
                 $product = $collection->addIdFilter($id)->getFirstItem();
-                $isInStock = $this->stockItemRepository->get($id)->getIsInStock();
+                $isInStock = false;
```

Comment:
Hey!
If stockItemRepository fails to get stock status by product id is it really means that this product is out of stock?

Reply:
@AleksandrsKondratjevs Technically yes. If we are unable to get the product, how can we verify the stock?